src/style.css

```diff
@@ -1 +1,114 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src: url(./fonts/Avenir-Book.ttf) format('truetype');
+}
+
+@font-face {
+  font-family: Avenir;
+  font-weight: 600;
+  src: url(./fonts/Avenir-Heavy.ttf) format('truetype');
+}
+
+* {
+  box-sizing: border-box;
+}
```

Comment:
Avoid using the universal selector '*' for setting box-sizing as it impacts performance. Apply box-sizing to specific elements as needed.